README.md

```diff
@@ -1,6 +1,6 @@
-# SUSE Console for SAP Applications v2
+# Trento
 
-A cloud-native, web application to manage OS-related tasks for SAP Applications.
+An open cloud-native, web console for managing SAP workloads on Linux.
```

Comment:
this conflicts with #12, maybe we could leave these diff lines out from this PR?

Reply:
I have rebased this now on top of #12 after merging that one.